go.mod

```diff
@@ -153,3 +153,7 @@ require (
 	modernc.org/strutil v1.2.0 // indirect
 	modernc.org/token v1.1.0 // indirect
 )
+
+replace go.temporal.io/api v1.34.0 => go.temporal.io/api v1.32.1
```

Comment:
What's the reason for downgrading `api`? This is the cause of https://github.com/temporalio/cli/issues/607, because it reintroduces a bug fixed in `api v1.34.0`: https://github.com/temporalio/api-go/pull/159

(I don't think `replace` statements should be committed; it makes it easy to look at the top of the file and reach an incorrect conclusion about what version it's on.)

Reply:
I'm fixing this by having `cli` use a cherry-pick branch of `api` for now: https://github.com/temporalio/cli/pull/617

Reply:
I had to downgrade the api with redirect because the go sdk release moved to a version of the api that was incompatible with the server one (the embedded server would not compile with that api), but luckily the sdk did not really need the new api , so I could get it to compile with the old one with the redirect...

Go sdk tends to upgrade to latest api in most releases, but server is slower, and the embedded server shares api with the sdk...